README.md

```diff
@@ -12,8 +12,22 @@ Avail is a data availability focused blockchain. Here is a list of GitHub repo t
 | [Reference Document](https://github.com/maticnetwork/data-availability/blob/master/reference%20document/Data%20Availability%20-%20Reference%20Document.pdf)     | Document containing the overall rationale, design decisions and theoretical backgrounds                              |
 | [Avail Node](https://github.com/maticnetwork/avail)                             | Avail node implementation repo. Built using Substrate.              |
 | [Light Client](https://github.com/maticnetwork/avail-light)                     | Light client designed to check data availability proofs on Avail.   |
-| [Explorer](https://github.com/maticnetwork/apps)                                | Avail explorer implementation repo. Built using Polkadot apps.      |
+| [Explorer](https://github.com/maticnetwork/avail-apps)                                | Avail explorer implementation repo. Built using Polkadot apps.      |
 | [Documentation](https://docs.polygon.technology/docs/avail/introduction/what-is-avail/) | Avail documentation hub.                                    |
 | [Tests](https://github.com/maticnetwork/avail-test)                             | End to end tests for Avail.                                         |
 
 Suggestions and feedback welcome.
+
+## Avail Architecture
+
+There are two main components in the Avail network:
+- [Avail Node](https://github.com/maticnetwork/avail): This forms the blockchain layer of Avail. It is built using Substrate, with modifications to specialize it for data availability. Apart from basic blockchain operations like balance transfer, validator staking, slashing and governance, it also contains methods for applications to submit data blobs. This network is secured by GRANDPA+BABE consensus engine, under the assumption that 2/3+ of the validator set is honest.
+- [Avail Light Client](https://github.com/maticnetwork/avail-light): The light clients (LC) form an overlay P2P network. Individually, all LCs subscribe to (finalized) headers of the chain. It performs Data Availability Sampling (DAS) to gain high confidence of DA. The LCs also host the sampled data on the P2P so that together the P2P network can have the entire data in a highly available manner.
```

Comment:
```suggestion
- [Avail Light Client](https://github.com/maticnetwork/avail-light): The light clients (LCs) form an overlay P2P network. Individually, all LCs subscribe to the (finalized) headers of the chain. The LCs perform Data Availability Sampling (DAS) to gain high confidence of DA. The LCs also host the sampled data on the P2P so that the whole P2P network can provide the entire data in a highly available manner.
```